### PR TITLE
Update phase-3-bootstrap-prompt.md

### PR DESCRIPTION
**Pull request by aedward8**
As this is my first time using bootstrap I think it would be helpful to really walkthrough the steps and be explicit that <div class=”row”> needs to be added

### CODE REVIEW

**Comment by alodahl** [comment]
@aedward8 could you please add a link to your description, and the exact section / line numbers that need to be changed?

**Comment by alodahl** [comment]
@aedward8 are you requesting that we make an issue about this, or is all you want to change included in this PR?

File: projects/recipe-page/phase-3-bootstrap-prompt.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ Save the file, then refresh `index.html` in Chrome by pressing `<COMMAND> + r`.
 
 Do you remember how to tell Bootstrap to allow a div to take up a certain number of columns? On a small screen, we usually want divs to take up all 12 columns so things are as large as possible. Let's add the `col-xs-12` class to the divs that currently have the `top`, `left` and `right` classes. (Remember, elements may have more than one class!) Here's an abbreviated example:
 
+Don't forget to add your <div class=”row”> before your column <div>
```
Comment (alodahl) [inline projects/recipe-page/phase-3-bootstrap-prompt.md:97]: is this a note, or part of the code? Putting it after the "Here's an example:" is a little confusing -  Maybe add it to the other "remember" statement in the parenthesis just above?

**Review by alodahl — CHANGES_REQUESTED** [review]

File: projects/recipe-page/phase-3-bootstrap-prompt.md (hunk 1 above)
Comment (aedward8) [reply projects/recipe-page/phase-3-bootstrap-prompt.md:97]: Yes I think that suggestion will work! To just add it to the "remember statement"

**Review by alodahl — APPROVED** [review]

**Pull request merged**